src/demos/calcite-icon.html

```diff
@@ -61,12 +61,10 @@ <h2>Dark theme</h2>
     <calcite-icon icon="camera" theme="dark"></calcite-icon>
   </div>
 
-  <h2>Flips icon in RTL when mirrored</h2>
-  <div dir="rtl">
```

Comment:
Is this demo published anywhere?

Reply:
The dev demos are available if you run the project locally - the `mirrored` prop is available as a knob in the storybook (which gets updated when each release goes out - https://esri.github.io/calcite-components/).

Reply:
Cool thanks. The `mirrored` prop knob does not work in storybook for me. Trying here: https://esri.github.io/calcite-components/?path=/story/icon--rtl

![Aug-11-2020 15-58-58](https://user-images.githubusercontent.com/8754/89957176-9dd4fc00-dbeb-11ea-9247-f32748daa104.gif)

Reply:
Yeah, currently it's broken on the link I provided - when this PR is merged and the next release goes out, it will get updated.

Reply:
In this example, why not use the right arrow glyph? Maybe the prop needs to be more explicit, but to me, mirrored is "this should flip in RTL" not "flip this icon".